drivers/timer/xlnx_psttc_timer.c

```diff
@@ -195,3 +195,8 @@ uint32_t sys_clock_cycle_get_32(void)
 	/* Return the current counter value */
 	return read_count();
 }
+
+uint64_t sys_clock_cycle_get_64(void)
+{
+	return 0;
```

Comment:
@stephanosio - would be great to get your input here. It seems there is no 64-bit timer register on this platform (i.e. the lptimer).

For software emulation, I would like to increment a 32-bit "top half" each time a 32-bit cycle counter overflows (assuming a 1:1 ratio), shift and then OR it with the regular 32-bit value.

In this driver, where is the best location to increment the 32-bit "top half"?